pkgs/development/python-modules/annotated-types/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hatchling
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "annotated-types";
+  version = "0.5.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "annotated-types";
+    repo = "annotated-types";
+    rev = "v${version}";
+    hash = "sha256-zCsWfJ8BQuov8FN+hlm9XBKWAAQ/KHPK/x024A8k2kE=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+  ];
+
+  propagatedBuildInputs = [
```

```suggestion
  propagatedBuildInputs = lib.optionals (pythonOlder "3.9") [
```
https://github.com/annotated-types/annotated-types/blob/0ad1bd8ea0323f270e5bb69f62235246ff80789e/pyproject.toml#L32

---

Thank you! Noted!